src/hivpy/sexual_behaviour.py

```diff
@@ -313,38 +314,39 @@ def balance_ltp_factor(self, population):
         else:
             return (1, 1)
 
-    def start_new_ltp(self, sex, age, size):
+    def start_new_ltp(self, sex, age_group, size):
         # TODO add alterations for new diagnosis
         ltp_rands = rng.random(size=size)
         rate_modifier = 1
-        if (35 <= age < 45):
+        if age_group == 1:
             rate_modifier = 0.5
-        elif (45 <= age < 55):
+        elif age_group == 2:
             rate_modifier = 1/3
-        elif (55 <= age):
+        elif age_group == 3:
             rate_modifier = 0.2
         new_relationship = ltp_rands < (self.new_ltp_rate
                                         * rate_modifier
                                         * self.ltp_balance_factor[sex])
         return new_relationship
 
-    def new_ltp_longevity(self, age, sex, size):
+    def new_ltp_longevity(self, age_group, size):
         longevity = np.zeros(size)  # each new relationship needs a longevity
         longevity_rands = rng.random(size)
-        if (age < 45):
-            longevity = ((longevity_rands < 0.3) * 1 +
-                         ((0.3 <= longevity_rands) & (longevity_rands < 0.6)) * 2 +
-                         (0.6 <= longevity_rands) * 3)
-        elif (age < 55):
-            longevity = ((longevity_rands < 0.3) * 1 +
-                         ((0.3 <= longevity_rands) & (longevity_rands < 0.8)) * 2 +
-                         (0.8 <= longevity_rands) * 3)
+        thresholds = [0, 0]
```

Comment:
This default value is always overwritten because all cases are covered below; I just really don't like the thing of declaring variables inside conditional logic which then need to be used outside of it